WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionHexCopiedViewModel.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using ReactiveUI;
+using WalletWasabi.Blockchain.Analysis.Clustering;
+using WalletWasabi.Blockchain.Keys;
+using WalletWasabi.Fluent.ViewModels.Navigation;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets.Send;
+
+public partial class TransactionHexCopiedViewModel : RoutableViewModel
+{
+	public string Caption { get; } = "The transaction hex was copied but not broadcast.";
+	public override string Title { get; protected set; } = "Transaction Hex Copied!";
+
+	[AutoNotify] private string _nextButtonText;
+	[AutoNotify] private string _cancelButtonText;
+
+	private readonly Wallet _wallet;
+	private readonly Dictionary<HdPubKey, LabelsArray> _hdPubKeysWithNewLabels;
+
+
+
+	private TransactionHexCopiedViewModel(Wallet wallet,
+		Dictionary<WalletWasabi.Blockchain.Keys.HdPubKey, WalletWasabi.Blockchain.Analysis.Clustering.LabelsArray> hdPubKeysWithNewLabels)
+	{
+		_wallet = wallet;
+		_hdPubKeysWithNewLabels = hdPubKeysWithNewLabels;
+
+		NextCommand = ReactiveCommand.Create(OnNext);
+		_nextButtonText = "Will broadcast";
```

Comment:
this is not so clear what it does.
feels like `Will broadcast` will broadcast the tx automatically.
or what the difference is in general is not clear

Reply:
I tried to use just the success screen that disappears, same as the traditional send, and it was not good because it gave the impression that the Hex has been broadcasted, so I had to add the line of text. So I can believe I have 2 options:
- Try to rework the wording somehow
- Forfeit the optimization, only have a `Done` button, and always call `UpdateUsedHdPubKeysLabels`

Reply:
I think always updating the `HdPubKey` is bad.
I rephrased the green button to "Broadcast Manually"
I added a small explanation stating to go back or cancel if there is no plan to broadcast manually
I also added a feature to rollback the clipboard to what it was before if the user goes back or cancels.

WDYT?

Reply:
yes this is better.
the `broadcast manually` button has big spacing after _manually_